### PR TITLE
Store Planet Descriptions in std::map

### PR DESCRIPTION
**Pull request by Brett208**
I think this will be easier to maintain in the long run than current form (which was commented as needing cleaned up). It would be nice to push into an xml file eventually.

### CODE REVIEW

File: OPHD/States/PlanetSelectState.cpp [hunk 1]
```diff
@@ -239,12 +239,14 @@ void PlanetSelectState::onMousePlanetEnter()
...
 		{
-			if (mPlanets[i]->attributes().type == Planet::PlanetType::Ganymede) { mPlanetDescription.text(constants::PLANET_DESCRIPTION_GANYMEDE); }
-			if (mPlanets[i]->attributes().type == Planet::PlanetType::Mars) { mPlanetDescription.text(constants::PLANET_DESCRIPTION_MARS); }
-			if (mPlanets[i]->attributes().type == Planet::PlanetType::Mercury) { mPlanetDescription.text(constants::PLANET_DESCRIPTION_MERCURY); }
+			auto iterator = constants::PLANET_DESCRIPTIONS.find(mPlanets[i]->attributes().type);
+
+			if (iterator != constants::PLANET_DESCRIPTIONS.end())
+			{
+				mPlanetDescription.text(iterator->second);
+			}
```
Comment (DanRStevens) [inline OPHD/States/PlanetSelectState.cpp:249]: You can potentially shorten this to:
```cpp
mPlanetDescription.text(constants::PLANET_DESCRIPTIONS.at(mPlanets[i]->attributes().type));
```

If the item does not exist in the `std::map`, the `at` method will `throw` and exception. Assuming you want an exception here of course.

Comment (Brett208) [reply OPHD/States/PlanetSelectState.cpp:249]: Ah right, I forgot about the `.at` function. I like the idea of adding an exception message.
